.zenodo.json

```diff
@@ -87,6 +105,21 @@
         {
             "name": "Roy Smart"
         },
+        {
+            "name": "Mihail Bankov"
```

Comment:
@mihailbankov Do you have an affiliation and orcid you would like us to include here?